shared/constants/swaps.js

```diff
@@ -42,6 +43,12 @@ export const TEST_ETH_SWAPS_TOKEN_OBJECT = {
   iconUrl: TEST_ETH_TOKEN_IMAGE_URL,
 };
 
+export const SWAPS_CHAIN_ID_TO_NETWORK_NAME_MAP = {
+  [MAINNET_CHAIN_ID]: 'Ethereum',
```

Comment:
These strings should be localized. "Test" probably isn't a very good name in all locales for example.

Reply:
I believe that network names should be the same across all languages. I like to think of it as names of companies.. we also don't translate them. 'Test' is a special one just for testing purposes and I'm open to changing it if there is something better. I thought that devs are used to hear Test Net(work), so that's why I chose "Test".

Reply:
I've changed it to "Testnet", because I believe that some translations don't use the word "network" after our dynamic value, so it would just say something like: "The network fee covers the cost of processing your swap and storing it on the Test".